velox/exec/tests/utils/QueryAssertions.cpp

```diff
@@ -914,6 +924,7 @@ bool assertEqualResults(
     const std::vector<RowVectorPtr>& expected,
     const std::vector<RowVectorPtr>& actual) {
   MaterializedRowMultiset expectedRows;
+  const TypePtr type = expected.at(0)->type();
```

Comment:
check that expected.size() > 0?

Reply:
Any specific message needs to be printed on checking?

Reply:
Fixed.

Reply:
Can be `VELOX_CHECK_GT`

Reply:
Fixed.